hollow/src/main/java/com/netflix/hollow/api/producer/validation/RecordCountVarianceValidator.java

```diff
@@ -93,9 +111,12 @@ public ValidationResult onValidate(ReadState readState) {
         vrb.detail(ACTUAL_CHANGE_PERCENT_NAME, actualChangePercent);
 
         if (Float.compare(actualChangePercent, allowableVariancePercent) > 0) {
-            String message = String.format(FAILED_RECORD_COUNT_VALIDATION, typeName, actualChangePercent,
-                    allowableVariancePercent);
-            return vrb.failed(message);
+            if(!onlyValidateDropsInCardinality || latestCardinality < previousCardinality) {
```

Comment:
Just a minor one : When would this flag value be used as false? could you add a test case where onlyValidateDropsInCardinality is false. The test cases you've added test the changes in cardinality but not this flag as such.

Reply:
Just to add : existing test cases should suffice for testing false value. So no need. I just wanted to clarify if there would be a case where you'd explicitly pass this flag value as false (apart from the default case).

Reply:
I would never choose to use the three argument constructor AND pass false for the `onlyValidateDropsInCardinality` parameter. I chose to override the 2 argument constructor so we don't break existing integrations when they upgrade.

Reply:
Okay. That default part I get. I was wondering if you were going to use it for local testing or some other case which I was missing.

Reply:
Changes look good. Merging this.